tests/qchem/of_tests/test_molecular_hamiltonian.py

```diff
@@ -401,6 +666,41 @@ def test_real_hamiltonian(symbols, geometry, method, args, tmpdir):
     assert np.isrealobj(hamiltonian.terms()[0])
 
 
+@pytest.mark.parametrize(
+    ("method", "args"),
+    [
+        (
+            "pyscf",
```

Comment:
These actually require new lines, or is the `black` doing something weird? 🤔

Reply:
Black does this whenever you have a trailing comma at the end
So ("method, args") -> no Change,
("method, args,") ->
`(`
`   "method",`
`    "args", `
 ` )`